src/test.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="utf-8">
     <title><%= htmlWebpackPlugin.options.title %></title>
-    <link href="//fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
+    <link href="http://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
```

Comment:
lack of protocol makes the browser use the current protocol from the address bar.

Was there a problem with this?

Reply:
It only didn't in the `file:\\\' protocol. I fixed it not wondering if we'd support the `file:///` environment.

Reply:
Ha, didn't think about that 😛